allensdk/brain_observatory/behavior/behavior_project_cache/project_metadata_writer/behavior_project_metadata_writer.py

```diff
@@ -0,0 +1,229 @@
+import argschema
+import os
+
+import pandas as pd
+from pathlib import Path
+
+import allensdk
+from allensdk.brain_observatory.behavior.behavior_project_cache import \
+    VisualBehaviorOphysProjectCache
+from allensdk.brain_observatory.behavior.behavior_project_cache.project_metadata_writer.schemas import (  # noqa: E501
+    BehaviorOphysMetadataInputSchema
+)
+from allensdk.brain_observatory.data_release_utils.metadata_utils.id_generator import (  # noqa: E501
+    FileIDGenerator,
+)
+from allensdk.brain_observatory.data_release_utils.metadata_utils.utils import (  # noqa: E501
+    add_file_paths_to_metadata_table,
+)
+
+#########
+# These columns should be dropped from external-facing metadata
+#########
+SESSION_SUPPRESS = (
+    'donor_id',
+    'foraging_id',
+    'session_name',
+    'specimen_id'
+)
+OPHYS_EXPERIMENTS_SUPPRESS = SESSION_SUPPRESS + (
+    'behavior_session_uuid',
+    'published_at',
+    'isi_experiment_id'
+)
+OPHYS_EXPERIMENTS_SUPPRESS_FINAL = [
+    'container_workflow_state',
+    'experiment_workflow_state']
+#########
+
+OUTPUT_METADATA_FILENAMES = {
+    'behavior_session_table': 'behavior_session_table.csv',
+    'ophys_session_table': 'ophys_session_table.csv',
+    'ophys_experiment_table': 'ophys_experiment_table.csv',
+    'ophys_cells_table': 'ophys_cells_table.csv'
+}
+
+
+class BehaviorProjectMetadataWriter(argschema.ArgSchemaParser):
+    """Class to write project-level metadata to csv"""
+
+    default_schema = BehaviorOphysMetadataInputSchema
+
+    def run(self):
+        """Create metadata tables and add file paths/ids.
+
+        Specifying directories will attempt to find appropriate NWB files to
```

Comment:
specifying directories where?

Reply:
Removed as that was referring to the previous iteration of this ticket.